bitcoin/CHANGELOG.md

```diff
@@ -57,6 +57,8 @@ This release is big, to help users upgrade we wrote a blog post, please see http
      contains a `LockTime` but is `Ord`, we have manually sorted the locktimes based on
      their consensus encoding. This ordering is somewhat arbitrary -- there is no total
      ordering on locktimes since they may be measured in either blocks or seconds.
+  - [Remove TxOut default trait](https://github.com/rust-bitcoin/pull/1838). The behavior
```

Comment:
nit:
```suggestion
  - [Remove TxOut default implementation](https://github.com/rust-bitcoin/pull/1838). The behavior
```

Reply:
Actually @yancyribbens could you take this suggestion, and capitalize `Default`?

Reply:
Thanks, this is now corrected.